client/src/lib/utils/assert.ts

```diff
@@ -0,0 +1,5 @@
+export function assert(condition: unknown): asserts condition {
+    if (!condition) {
+        throw new Error('Assertion failed.');
+    }
```

Comment:
No need for the braces here.
```suggestion
    if (!condition) throw new Error('Assertion failed.');
```